lib/lbt/bundle/Builder.js

```diff
@@ -200,23 +215,35 @@ class BundleBuilder {
 		}
 	}
 
+	writeWithSourceMap(content) {
+		const transientSourceName = `${path.posix.basename(this._bundleName)}?inline-${this._inlineContentCounter++}`;
```

Comment:
Is this now the final name, or do we have some other suggestions?
What about `?bundled-code-${i}` ?

Reply:
Webpack uses an own protocol `webpack://` for sources that are part of the map (sourcesContent).
Built-in code is references e.g. as `webpack://<project-name>/webpack/bootstrap`.

Reply:
Also see https://webpack.js.org/configuration/output/#outputdevtoolmodulefilenametemplate

Reply:
I guess if we use a different protocol for those sources, they won't appear next to the bundle in the source tree of the dev tools?

Reply:
@codeworrior do you have any preference?

Reply:
I guess I like 7 and 8 most, but not so much that I would say: go for it!

The good thing is, we can change the names, if we want. They're local to the source map, aren't they? 

I also had some doubts regarding the own protocol. Now that Matthias mentioned it, I remember that we encountered it in real life already. See internal Snippet #93307 (it shows a blank page - open dev tools and check the sources tree).

Chrome shows the webpack:// sources perfectly fine. Maybe the URL does not matter when there's a sources entry in the source map.

Reply:
I also liked 7 and 8 the most. I tried 8 but then the name got truncated in a bad way. `bundle-code` seems to work pretty well:
![Screenshot 2022-02-23 at 10 23 28](https://user-images.githubusercontent.com/1589939/155292039-e043bf47-b0f9-431d-80ec-860666953bb0.png)
![Screenshot 2022-02-23 at 10 26 33](https://user-images.githubusercontent.com/1589939/155292049-1aa0bc60-2287-48d2-b8e8-4ad064bca0e9.png)

Reply:
I just wanted to mention how webpack does it, but I'm not really in favor of it.

`bundle-code` sounds good 👍🏻